SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/LayeredLBStrategy.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "LoadBalancing/AbstractLBStrategy.h"
+
+#include "CoreMinimal.h"
+#include "Math/Box2D.h"
+#include "Math/Vector2D.h"
+
+#include "LayeredLBStrategy.generated.h"
+
+class SpatialVirtualWorkerTranslator;
+class UAbstractLockingPolicy;
+
+DECLARE_LOG_CATEGORY_EXTERN(LogLayeredLBStrategy, Log, All)
+
+USTRUCT()
+struct FLBLayerInfo
+{
+	GENERATED_BODY()
+
+	UPROPERTY()
+	FName Name;
+
+	UPROPERTY(EditAnywhere, Category = "Load Balancing")
+	TSubclassOf<UAbstractLBStrategy> LoadBalanceStrategy;
+
+	UPROPERTY(EditAnywhere, Category = "Load Balancing")
+	TSubclassOf<UAbstractLockingPolicy> LockingPolicy;
+
+
+	FLBLayerInfo() : Name(NAME_None)
```

Comment:
Perhaps you prefer constructors at the top of classes/ structs?

Reply:
This was copied from the FActorGroupInfo and they were in that order there. I wasn't sure if there was a convention of UPROPERTY values before constructors, but if there isn't, I'm happy to move it up.